tests/contrib/starlette/test_starlette.py

```diff
@@ -0,0 +1,266 @@
+import asyncio
+import nest_asyncio
```

Comment:
Is `nest_asyncio` necessary for the tests?

Reply:
The tests wouldn't work unless I called nest_asyncio.apply() at the end of all the testcases